components/micro-gateway-cli/src/main/java/org/wso2/apimgt/gateway/cli/utils/OpenAPICodegenUtils.java

```diff
@@ -702,11 +718,25 @@ private static void validateSingleResourceInterceptors(Operation operation, Stri
      */
     private static void setMgwAPISecurityAndScopes(ExtendedAPI api, OpenAPI openAPI) {
         String[] securitySchemasAndScopes = generateMgwSecuritySchemasAndScopes(openAPI.getSecurity());
+        if (openAPI.getExtensions() != null && !openAPI.getExtensions().isEmpty() &&
```

Comment:
`openAPI.getExtensions()` Keep this in a variable and refer it.

Reply:
Fixed. Thanks https://github.com/wso2/product-microgateway/pull/970/commits/c023abac6801c1fdbfbeb4795dac35a90ea8b045